targets/targets.json

```diff
@@ -1997,6 +2001,10 @@
             "FLASH",
             "MPU"
         ],
+        "overrides": {
+            "tickless-from-us-ticker": true,
```

Comment:
That switches the deep sleep off, so the gains from tickless are not amazing.

Reply:
This platform does not have an LPTICKER that matches the mbed requirements. What do you suggest?